src/views/Order/Summary.js

```diff
@@ -0,0 +1,107 @@
+import React, { Component } from 'react';
+// import Dashboard from '../Dashboard'
+// import { isAuthEmail, authenticateUser, isAuthUserType } from '../../services/Auth'
+import { Link } from 'react-router-dom';
+// import CustomerMap from '../Map/CustomerMap'
+import {
+  Button,
+  Row
+} from 'reactstrap';
+import Payment from '../../components/Payment';
+
+class Summary extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      modal: false
+    };
+    this.toggle = this.toggle.bind(this);
+  }
+
+  toggle() {
+    this.setState({
+      modal: !this.state.modal,
+    });
+  }
+
+  render() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected 'this' to be used by class method 'render'.](https://app.codacy.com/gh/BuildForSDG/Team-064-Frontend/pullRequest?prid=5703258)